acceptance/config/aio/options.rb

```diff
@@ -16,4 +16,7 @@
     'setup/common/050_Setup_Broker.rb',
     'setup/common/060_Setup_PCP_Client.rb',
   ],
+  :post_suite => [
+    'setup/common/099_Archive_Logs.rb',
```

Comment:
This perhaps should go into a different folder than 'setup'. I don't think there's a precedent for post-suite folder name in the puppet-agent repos. 'post_suite' perhaps.